examples/np/cem_cartpole.py

```diff
@@ -27,6 +27,8 @@ def run_task(snapshot_config, *_):
 
         baseline = LinearFeatureBaseline(env_spec=env.spec)
 
+        runner.initialize_tf_vars()
```

Comment:
Why is `runner.setup()` not working here?

Reply:
Because CategoricalMLPPolicy is a tf policy and requires variables before `setup()`.

Reply:
Hmm can you send me a permalink of the variables?

Reply:
So it is used here:
https://github.com/rlworkgroup/garage/blob/2c23600100250344bf0d9a36f89f47d5e944c9e7/src/garage/np/algos/cem.py#L63
We should probably do some refactoring.

Reply:
Yes please file an issue for this. I think we can just initialize it as `None` at the beginning and only set it when the training start.